compiler/ExtractBase.ml

```diff
@@ -1535,7 +1546,10 @@ let ctx_compute_trait_impl_name (ctx : extraction_ctx) (trait_decl : trait_decl)
   let name =
     let params = trait_impl.llbc_generics in
     let args = trait_impl.llbc_impl_trait.decl_generics in
-    let name = ctx_prepare_name trait_impl.span ctx trait_decl.llbc_name in
+    let name =
+      ctx_prepare_name trait_impl.item_meta.span ctx trait_decl.llbc_name
+    in
+    (* let name = rename_llbc_name trait_impl.item_meta name in  *)
```

Comment:
Commented code.
You are also ignoring the `rename` attribute here.

Reply:
You are still ignoring the `rename` attribute here. Please be careful when marking conversations as resolved.